governance/chair-transition.md

```diff
@@ -0,0 +1,62 @@
+# Chair transition checklist
+
+This checklist should be copied into a new issue for when a new chair(s) is needed.
+
+## Nomination
+
+This serves to augement the [Security TAG Chair proposal
+process](chair-proposal-process.md) with specific items that must be completed.
+
+* [ ] Nomination
+  * [ ] TOC-Liaison slack discussion and approval
+  * [ ] Co-chair & Tech Lead meeting announcement
+  * [ ] Security TAG Meeting announcement -- all regions
+  * [ ] TOC email list official nomination
+  * [ ] #tag-security slack message with a link to the email nomination
+  * [ ] TOC meeting presentation
+
+* [ ] Vote - official count recorded and posted
+
+## Transfer
+
+* [ ] Transfer Logisitics
+  * [ ] #tag-chairs slack channel - add new chair(s)
+  * [ ] tag-chairs mailing list - add new chair(s) 
+  * [ ] #tag-security-cochairs channel - add new chair(s)
+  * [ ] #tag-security-cochairs-emeritus - add outgoing chair(s)
+  * [ ] #tag-security-toc-liaison - add new chair(s)
+  * [ ] Co-chair meeting - add new co-chairs and add doc sharing
+  * [ ] If applicable - transfer ownership of co-chair meeting calendar invite
+    and tech lead calendar invite
+    * [ ] New meeting link will need created by new calendar owner if using
+      Google meet.  meet.new will create this, just copy into invite.
```

Comment:
we should probably at some point use zoom so we have transcriptions as well if we have to look back on it